lms/templates/course.html

```diff
@@ -6,34 +6,17 @@
 %>
 <%page args="course" />
 <article id="${course.id | h}" class="course">
```

Comment:
a nice touch here would be to add a ```role="region"``` to this ```article``` element with an ```aria-labelledby``` that references the ID of the ```<h2>``` that has the course title in it.  If you don't want to deal with IDs, you can use ```aria-label="${get_course_about_section(course, 'title')}"```  Which allows a screen reader user to choose whether or not they want to consume the child content of this "region" or to skip over it.